### PR TITLE
 Adds 5 New Bars To the Bar Randomizer (5 old) Branch Edition V2! 

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑 fluffe9911
add: 5 new bars that the regular bar can possibly become!
/🆑

Most of the Bars can be found here yogstation13/yogstation#3109

### CODE REVIEW

**Comment by Time-Green** [comment]
you ok man?

**Comment by ThatLing** [comment]
Delete corporate and xeno

File: tgstation.dme [hunk 1]
```diff
@@ -2744,4 +2744,5 @@
 #include "interface\menu.dm"
 #include "interface\stylesheet.dm"
 #include "interface\skin.dmf"
+#include "yogstation\code\game\objects\effects\decals\turfdecal.dm"
```
Comment (ThatLing) [inline tgstation.dme]: Remove

**Comment by Ktlwjec** [comment]
image
I just noticed this table isn't reinforced on the server please change xoxo

File: tgstation.dme (hunk 1 above)
Comment (ThatLing) [inline tgstation.dme]: Remove this from tgstation.dme

File: yogstation/code/datums/ruins/station.dm [hunk 2]
```diff
@@ -24,3 +24,48 @@
...
+	id = "bar_ragecage"
+	suffix = "bar_ragecage.dmm"
+	name = "Bar Ragecage"
+
+/datum/map_template/ruin/station/box/bar/space
+	id = "bar_space"
+	suffix = "bar_space.dmm"
+	name = "Bar Space"
+
+/datum/map_template/ruin/station/box/bar/xeno
```
Comment (ThatLing) [inline yogstation/code/datums/ruins/station.dm]: Remove xenobar, also isn't in

File: yogstation/code/datums/ruins/station.dm [hunk 3]
```diff
@@ -24,3 +24,48 @@
...
+	id = "bar_disco"
+	suffix = "bar_disco.dmm"
+	name = "Bar Disco"
+
+/datum/map_template/ruin/station/box/bar/purple
+	id = "bar_purple"
+	suffix = "bar_purple.dmm"
+	name = "Bar Purple"
+
+/datum/map_template/ruin/station/box/bar/ragecage
```
Comment (ThatLing) [inline yogstation/code/datums/ruins/station.dm]: This one isn't even in?

File: yogstation/code/datums/ruins/station.dm [hunk 4]
```diff
@@ -24,3 +24,48 @@
...
+	id = "bar_ragecage"
+	suffix = "bar_ragecage.dmm"
+	name = "Bar Ragecage"
+
+/datum/map_template/ruin/station/box/bar/space
```
Comment (ThatLing) [inline yogstation/code/datums/ruins/station.dm]: This one isn't in

File: yogstation/code/datums/ruins/station.dm [hunk 5]
```diff
@@ -24,3 +24,48 @@
...
+	id = "bar_casino"
+	suffix = "bar_casino.dmm"
+	name = "Bar Casino"
+
+/datum/map_template/ruin/station/box/bar/conveyor
+	id = "bar_conveyor"
+	suffix = "bar_conveyor.dmm"
+	name = "Bar Conveyor"
+
+/datum/map_template/ruin/station/box/bar/corporate
```
Comment (ThatLing) [inline yogstation/code/datums/ruins/station.dm]: This one isn't in

File: yogstation/code/game/objects/effects/landmarks.dm [hunk 6]
```diff
@@ -62,7 +62,7 @@
 	return TRUE
 
 /obj/effect/landmark/stationroom/box/bar
-	template_names = list("Bar Trek", "Bar Spacious", "Bar Box")
+	template_names = list("Bar Trek", "Bar Spacious", "Bar Box", "Bar Casino", "Bar Conveyor", "Bar Corporate", "Bar Diner", "Bar Disco", "Bar Purple", "Bar Ragecage", "Bar Space", "Bar Xeno")
```
Comment (ThatLing) [inline yogstation/code/game/objects/effects/landmarks.dm]: Again, some of these aren't in

File: yogstation/code/game/objects/structures/bar_stuff/bar_stuff.dm [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+
+
+/obj/machinery/computer/slot_machine/casino
+	name = "slot machine"
+	desc = "Gambling for the antisocial."
+	icon = 'yogstation/icons/obj/special_slot_machine.dmi'
+	icon_state = "slots1"
+
+/turf/open/floor/alien
```
Comment (ThatLing) [inline yogstation/code/game/objects/structures/bar_stuff/bar_stuff.dm]: Remove all this xenobar stuff, including icons

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
Decal dirs were changed in https://github.com/tgstation/tgstation/pull/42261, mirrored in https://github.com/yogstation13/Yogstation-TG/pull/4141. To fix your decals, update your branch to latest master then run the fixcorners.txt script by doing the following after opening command line in the tools/mapmerge2 folder:

python -m update_paths --map [path-for-the-map-you're-updating] [path-for-the-fixcorners-script,-will-end-in-tools/mapmerge2/map_scripts/fixcorners.txt

**Comment by fluffe9911** [comment]
>nich tells me to use the script to fix my decals
>I have none of the effected decals in any of my map prs

nich please

**Review by Amelia0010 — APPROVED** [review]
I just copy pasted that on all map PRs to make sure it wasn't bork :)

**Comment by Amelia0010** [comment]
I swear to god stop fucking deleting your repos you mongrel

**Pull request closed without merging**